### DISCUSSION TITLE
Self-host - Easy solve for email verification error

### DISCUSSION THREAD

**Discussion by sethmbhele** [plandex-ai/plandex#100] [General]
Hi Friends

Thank you so much for the effort that went into this early product; I have just got my setup working today and can't wait to really get stuck in.

Just one quick note, to do with ports the docker-compose file: I was getting the following error resolving auth when I was signing up: 

![Screen Shot 2024-04-24 at 14 38 13](https://github.com/plandex-ai/plandex/assets/4163455/004ceed6-09a8-4fd2-9bdb-fd52819553f3)

For me, it was an easy solve, added ports 80:80 to plandex-server in the compose file. Not sure if this breaks something in terms of good security practice in prod on my part; but running on my local lab, so not too worried. If this would help others possibly struggling with the same error, perhaps if you can consider this edit going forward?

Amazing work - will give more feedback as I go. So looking forward to it :)

**Comment by danenania** [comment]
@sethmbhele Thanks for noting this. I hope you don't mind if I move this over to a discussion. I'm looking forward to hearing your feedback!

**Reply by arty-hlr** [reply]
@danenania Unfortunately I have the same error with plandex behind a reverse proxy with TLS:
![image](https://github.com/user-attachments/assets/d966f89b-846d-44a9-bb44-a474ceb12a94)

Signing in worked on an ubuntu machine though, this is a mac, I'm not sure if that makes a difference?

**Reply by laur89** [reply]
> Unfortunately I have the same error with plandex behind a reverse proxy with TLS

I chose `https://mydomain.example.com` as a host that's behind TLS-terminating nginx proxy, and setup worked fine.
